src/lib/builtins.pl

```diff
@@ -1374,10 +1368,7 @@
        '$get_db_refs'(_, _, _, PIs),
        lists:member(Pred, PIs)
     ;  loader:strip_module(Pred, Module, UnqualifiedPred),
-       (  var(Module),
-          \+ functor(Pred, (:), 2)
-       ;  atom(Module)
-       ),
+       atom(Module),
```

Comment:
`atom/1` seems guaranteed qua (now improved) definition of `strip_module/3`.

Reply:
I see now that `atom/1` is not guaranteed, I remove the earlier comment, thank you a lot!